code/modules/cm_aliens/structures/trap.dm

```diff
@@ -217,6 +217,20 @@
 			to_chat(X, SPAN_XENONOTICE("[src] is occupied by a child."))
 			return XENO_NO_DELAY_ACTION
 
+	if(X.a_intent == INTENT_HARM)
```

Comment:
Please fix the single letter vars as you need to modify code. See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#variables

In VSC its very easy to do this, just ctrl+f, highlight the entire proc block, press the paragraph looking button to restrict search to selection, match case, and match word, and then dropdown to reveal replace.

Reply:
E.g. 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/9e3424bf-e3b3-4ab3-af11-eabb66a84043)

Reply:
Have the queen option for both of them, in separate toggles?

Reply:
Search for one of those options and you'll see what I mean. It's a single setting with 3 possible states.